FitVid/src/Components/Video/video.css

```diff
@@ -5,9 +5,27 @@
     cursor: pointer;
     background-color: var(--background-color-dark);
 }
-.videolisting-video{
+.videolisting-video-thumbnail{
+    object-fit: contain;
+}
+.videolisting-thumbnail-container{
+    position: relative;
+    width: "100%";
```

Comment:
why you have wrapped in double quotes ?? 
It's not needed . Remove it.

Reply:
Thanks, for pointing this out